evil-keybindings.el

```diff
@@ -58,6 +58,11 @@
 
 (eval-after-load 'dired
   '(progn
+     ;; support evil-jumps
```

Comment:
This belongs in `evil-integration.el`, and could use `mapc`.

Reply:
evil-jumps works fine for jumping around dired buffers without these lines. I don't know why it was needed last time. Maybe I was working on old version of evil that requires settings `:jump` for dired dired commands.

I have removed these lines. We may add :jump for dired later if it is found necessary.